sunbird/sample1.py

```diff
@@ -0,0 +1,5395 @@
+import imaplib
+import base64
+import os
+import time
+from configparser import ConfigParser, ExtendedInterpolation
+import wget
+import urllib
+import xlrd
+import uuid
+import csv
+from bson.objectid import ObjectId
+import json
+from datetime import datetime
+import requests
+from difflib import get_close_matches
+from requests import post, get, delete
+import sys
+import time
+import xlwt
+import xlutils
+from xlutils.copy import copy
+import shutil
+import re
+from xlrd import open_workbook
+from xlutils.copy import copy as xl_copy
+import logging
+import logging.handlers
+import time
+from logging.handlers import TimedRotatingFileHandler
+import xlsxwriter
+import argparse
+import sys
+from os import path
+import pandas as pd
+import openpyxl
+from openpyxl import Workbook
+from openpyxl.styles import Color, PatternFill, Font, Border
+from openpyxl.styles import colors
+from openpyxl.cell import Cell
+import gdown
+
+# get current working directory
+currentDirectory = os.getcwd()
+
+# Read config file 
+config = ConfigParser()
+config.read('common_config/config.ini')
+
+
+# email regex
+regex = "\"?([-a-zA-Z0-9.`?{}]+@\w+\.\w+)\"?"
+
+# Global variable declaration
+criteriaLookUp = dict()
+millisecond = None
+programNameInp = None
+environment = None
+observationId = None
+solutionName = None
+pointBasedValue = None
+entityType = None
+allow_multiple_submissions = None
+scopeEntityType = ""
+programName = None
+userEntity = None
+roles = ""
+mainRole = ""
+dictCritLookUp = {}
+isProgramnamePresent = None
+solutionLanguage = None
+keyWords = None
+entityTypeId = None
+solutionDescription = None
+creator = None
+dikshaLoginId = None
+criteriaName = None
+solutionId = None
+API_log = None
+listOfFoundRoles = []
+entityToUpload = None
+programID = None
+programExternalId = None
+programDescription = None
+criteriaLookUp = dict()
+themesSheetList = []
+themeRubricFileObj = dict()
+criteriaLevelsReport = False
+ecm_sections = dict()
+criteriaLevelsCount = 0
+numberOfResponses = 0
+criteriaIdNameDict = dict()
+criteriaLevels = list()
+matchedShikshalokamLoginId = None
+scopeEntities = []
+scopeRoles = []
+countImps = 0
+ecmToSection = dict()
+entitiesPGM = []
+entitiesPGMID = []
+solutionRolesArr = []
+startDateOfResource = None
+endDateOfResource = None
+startDateOfProgram = None
+endDateOfProgram = None
+rolesPGM =None
+solutionRolesArray = []
+solutionStartDate = ""
+solutionEndDate = ""
+projectCreator = ""
+orgIds = []
+OrgName = []
+ccRootOrgName = None
+ccRootOrgId  = None
+certificatetemplateid = None
+question_sequence_arr = []
+
+# function to map course to program
+
+
+# As per the discussion with products team, course is currently taken down from the product until further notice 
+def courseMapToProgram(accessToken, courseLink, parentFolder):
```

Comment:
As these functions are not used for the current implementation remove this function and the fn calls as well